keps/prod-readiness/sig-apps/2890.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2890
+beta:
```

Comment:
This is new feature so it has to go through full alpha->beta->GA cycle, not shortcuts.

Reply:
ok，i will modify KEP，firstly alpha，then beta，finally GA，thanks.